python-threatexchange/threatexchange/exchanges/signal_exchange_api.py

```diff
@@ -189,6 +193,8 @@ def naive_fetch_merge(
         """
         Merge the results of a fetch in-memory.
 
+        DEPRECATED: This method will be removed in version 2.x.x.
+
```

Comment:
We should keep this version around - it's marked final, and the argument to keep it around is on L198.

Instead, we should should try and tell if it's safe to change this method not to call fetch_value_merge. 

```suggestion
```